docs/sections/guides.md

```diff
@@ -129,15 +129,15 @@ In this particular example, these scripts depend on CURL and Python.
 Thankfully, docker images for these already exist, so we can make use 
 of them as follows:
 
-```yaml
+```hcl
 - uses: docker://byrnedo/alpine-curl:0.1.8
   args: ["scripts/download-data.sh"]
 
 - uses: docker://python:3.7
   args: ["./scripts/get_mean_by_group.py", "5"]
 
 - uses: docker://python:3.7
-  args: [
+  args [
```

Comment:
without the dots this should break, does it run fine on your end?

Reply:
weird that that got deleted. Don't know how that crept in.